### ISSUE TITLE
The Evolution of JavaScript

### ISSUE THREAD

**Issue by CaptainStark3k** [Grimm-s-Alchemy-Chamber/hackers_hive#70]
# Title Of Blog Post : The Evolution of JavaScript

**Comment by shreyanshig** [comment]
Can you please assign this task to me?

**Comment by deep-Ak-47** [comment]
yeah sure you can work on it and please refer to contributing guidelines.

**Comment by shreyanshig** [comment]
Hi, am I required to create a simple .txt file or a webpage describing the evolution of JavaScript?

**Comment by shreyanshig** [comment]
Hey i have completed it and made a PR, please review and merge it as i have
worked quite alot in it
